python/lance/tests/test_api.py

```diff
@@ -16,10 +16,10 @@
 from pathlib import Path
```

Comment:
Changes in this whole file is just formatting right?

Reply:
yes, that's right.